lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,46 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  devServer:                  # https://webpack.js.org/configuration/dev-server/
+    host: localhost           # Dev server host
+    port: 8080
+    https: false
+
+  paths:                       # ~ = Rails.root
+    source: app/javascript     # ~/:source
+    entry: packs               # ~/:source/:entry
+    output: packs              # ~/public/:output
+    manifest: manifest.json    # ~/public/:output/:manifest
+    config: config/webpack     # ~/:config
+
+    extensions:
+      - .coffee
+      - .js
+      - .jsx
+      - .ts
+      - .vue
+      - .sass
+      - .scss
+      - .css
+      - .png
+      - .svg
+      - .gif
+      - .jpeg
+      - .jpg
+
+development:
+  <<: *default
+
+test:
+  <<: *default
+
+  paths:
+    manifest: manifest-test.json
+
+production:
+  <<: *default
+
+  devServer:
+    host: example.com
+    port:
+    https: true
```

Comment:
Ahh yes my bad, mixed it up with this one - https://github.com/rails/webpacker/issues/176. Changed it to `0.0.0.0`